devtools/docker-compose.yml

```diff
@@ -177,34 +166,33 @@ services:
     build:
       context: ../src/backend
       dockerfile: Dockerfile.imap-worker
-    image: caliopen_imap_worker
```

Comment:
There isn't anything that should break with the change, I just like having images built with a more decent name.